spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Output.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.mkldnn
+
+import com.intel.analytics.bigdl.mkl.{Memory, MklDnn}
+import com.intel.analytics.bigdl.nn.abstractnn.Activity
+import com.intel.analytics.bigdl.tensor.{DnnTensor, Tensor}
+
+/**
+ * Convert output to user defined layout and appoint gradOutput layout
+ * @param outputLayOut output memory layout
+ * @param gradOutputLayout gradoutput memory layout,
+ *                         if it is -1, that means gradOutputLayout same with outputLayOut
```

Comment:
same on what ? I think you should clarify it to make it more clear to user,  is here you mean layout format right ?

Reply:
ok, changed to more clear  comment